zfs-replicate.sh

```diff
@@ -479,19 +495,23 @@ init() {
 	echo "Replication mode is not set. Please set the MODE variable to PUSH or PULL."
     	exit_error
     fi
+    if [ -z "${REPLICATE_SETS}" ] || [ "${REPLICATE_SETS}" == "localpool/localdataset:remotepool/remotedataset" ]; then
+    	echo "REPLICATE_SETS is not set properly. Please set it. See config.sample.sh file for examples."
+     	exit_error
+    fi
     if [ $SNAP_KEEP -lt 2 ]; then
         printf "ERROR: You must keep at least 2 snaps for incremental sending.\n"
         printf "Please check the setting of 'SNAP_KEEP' in the script.\n"
         exit_error
     fi
     ## check remote health
-    printf "Checking remote system...\n"
+    echo "Checking remote system..."
```

Comment:
Same here and the other changes, these should be consistent.

Reply:
You do have a point there.